dev/aarch64_opt/src/Makefile

```diff
@@ -0,0 +1,40 @@
+# SPDX-License-Identifier: Apache-2.0
+
+######
+# To run
+#
+# NOT IN THE NIX ENV
```

Comment:
Could you please add a github action to this PR which exemplifies and tests this flow?

More concretely: Add a `setup_slothy` action which, similar to (say) `setup_apt` setups up SLOTHY assuming a standard ubuntu environment. Then you can run this action from an `ubuntu-latest` runner as the first step in a new SLOTHY workflow in `.github/workflows/slothy.yml`.

If you have a look at the existing YML files, I'm sure you'll figure it out.